db/createItem.go

```diff
@@ -12,9 +12,12 @@ func (d *dynamoDB) CreateItem(listID string, name string) (*data.Item, error) {
 	itemID := d.generateID()
 
 	item := &data.Item{
-		ListID: listID,
-		ID:     itemID,
-		Name:   name,
+		ItemKey: data.ItemKey{
+			ListID: listID,
+			ID:     itemID,
+		},
+		Name:        name,
+		IsCompleted: false,
```

Comment:
When an item is first created it is not completed, we could support it being complete straight away by adding it to the POST body, but I can't think of a use-case for this

Reply:
I like what you have atm - always set as false